tests/jest/helpers/main.js

```diff
@@ -0,0 +1,221 @@
+const ChatkitServer = require("@pusher/chatkit-server").default
+const uuid = require("uuid/v4")
+
+const config = require("./config/production")
+
+async function defaultBeforeAll(roleName) {
+  page.on("console", async msg => {
+    const argsWithRichErrors = await Promise.all(
+      msg
+        .args()
+        .map(arg =>
+          arg
+            .executionContext()
+            .evaluate(arg => (arg instanceof Error ? arg.message : arg), arg),
+        ),
+    )
+    console.log(...argsWithRichErrors)
+  })
```

Comment:
probably want the `page.on("error")` and `page.on("pageerror")` from `puppeteer-run` too